src/service/NylasService/NylasService.ts

```diff
@@ -65,6 +65,49 @@ export class NylasService extends APIClient {
         });
     }
 
+    /**
+     * Returns an attachment metadata using the nylas api
+     * @param emailAccount the email to be sending from
+     * @param attachmentId
+     */
+    async getAttachmentMetadata(emailAccount: string, attachmentId: string): Promise<AxiosResponse> {
```

Comment:
Endpoints which are fetching things have to be prefixed with `fetch` 
```suggestion
    async fetchAttachmentMetadata(emailAccount: string, attachmentId: string): Promise<AxiosResponse> {
```